operator_ui/src/PrivateRoute.test.js

```diff
@@ -24,18 +27,17 @@ const mountAuthenticatedApp = (store) =>
   )
 
 describe('PrivateRoute', () => {
-  it('redirects when user is NOT autheticated', () => {
+  it('redirects when user is NOT autheticated', async () => {
     const state = { authentication: { allowed: false } }
-    const wrapper = mountAuthenticatedApp(mockStore(state))
-    expect(wrapper.find(AuthenticatedApp).props().location.pathname).toBe(
-      '/signin',
-    )
+    mountAuthenticatedApp(mockStore(state))
```

Comment:
```suggestion
    const { getByText } = mountAuthenticatedApp(mockStore(state))
```
1. I think `getBy*` should be preferred if we aren't doing async stuff (looks like that's the case here).
2. `render` returns methods that can be used. I think that should be preferred as `screen` uses a global.

Reply:
Discussed in Slack and #2 should be ignored. Seems like using `screen` is best practice https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen